_includes/game-card.html

```diff
@@ -10,11 +10,11 @@
       style="background-image: url('{{ include.icon }}');"
     ></div>
   </div>
-  <p class="text-accent font-bold text-base md: lg:text-[20px] py-5">
+  <p class="accent-with-hover font-bold text-base md: lg:text-[20px] py-5">
     {{ include.title }}
   </p>
   <p
-    class="text-center w-10/12 leading-tight font-medium max-w-noice md:max-w-none lg:max-w-none"
+    class="text-with-hover text-center w-10/12 leading-tight font-medium max-w-noice md:max-w-none lg:max-w-none"
```

Comment:
these are handled as "exceptions" because the whole game card is a link
everywhere else it should just work semantically